config/config_tests.xml

```diff
@@ -244,7 +244,12 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <BFBFLAG>TRUE</BFBFLAG>
     <STOP_OPTION>ndays</STOP_OPTION>
     <STOP_N>11</STOP_N>
+    <REST_N>$STOP_N / 2 + 1</REST_N>
```

Comment:
Also, more generally, I find it confusing if critical test-specific settings are done in config_tests.xml, because I would expect to be able to understand the critical aspects of how a test operates by looking at that test's python file. But I realize that other people have different opinions on that...